src/Web/Controllers/AccountController.cs

```diff
@@ -237,7 +238,9 @@ private async Task<IActionResult> CallCustomAssessmentApi(CustomAssessmentViewMo
             var correlationId = _fraudProtectionService.NewCorrelationId;
             var payload = model.Payload
                 .Replace("@deviceFingerprintingId", model.DeviceFingerPrinting.SessionId)
-                .Replace("@deviceIpAddress", _contextAccessor.HttpContext.Connection.RemoteIpAddress.MapToIPv4().ToString());
+                .Replace("@deviceIpAddress", _contextAccessor.HttpContext.Connection.RemoteIpAddress.MapToIPv4().ToString())
+                .Replace("@guid", Guid.NewGuid().ToString())
+                .Replace("@timestampNow", DateTime.UtcNow.ToString("yyyy-MM-ddTHH:mm:ssZ", CultureInfo.InvariantCulture));
```

Comment:
should this use the shortcut format string or does that not work?
https://learn.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings#the-round-trip-o-o-format-specifier

Reply:
Thanks; yes this shortcut works as well (there's a couple extra digits of precision for the seconds but I think that's fine)